src/amo/components/Collection/index.js

```diff
@@ -234,6 +235,7 @@ export class CollectionBase extends React.Component<Props> {
           <AddonsCard
             addonInstallSource={INSTALL_SOURCE_COLLECTION}
             addons={addons}
+            forCollectionMaintenance={editing}
```

Comment:
Yeah, seeing this here... that prop should just be called `editing` or `isEditing`.

Reply:
The thing is, I'll be implementing this for "creating" as well very soon, so I was thinking ahead. It doesn't matter if we're editing or creating, in both cases we're "maintaining" and this will be a `true`. So if I call it `editing` now I'll have to change it later. Does that make sense?

Reply:
I still feel like `editing` is a better fit for creating than `maintaining`–I think it's clearer and easier to understand. 🤷‍♂️

Reply:
I don't agree, but it's not that important to me. I'll change it, and if anyone complains when `editing` is also used for `creating` I'll point them to this comment. ;-)